core/templates.yml

```diff
@@ -89,6 +89,8 @@ keys:
             exr: exr
             jpg: jpeg
             tiff: tiff
+            png: PNG
```

Comment:
why are these uppercase and the others lower? can't we be consistent?

Reply:
Ah! my bad will fix this.